tests/coverage

```diff
@@ -1 +1 @@
-63.0
\ No newline at end of file
+67.0
```

Comment:
Please swap the word order in your commit message, adjective before noun, i.e. "coverage: update for current higher coverage"